app/liquid_tags/email_signup_tag.rb

```diff
@@ -0,0 +1,9 @@
+class EmailSignupTag < LiquidTagBase
```

Comment:
This entire liquid tag will be built in the next PR. For now, it's "empty" so that
```ruby
liquid_tags.include?(EmailSignupTag)
```
doesn't blow up in the controller with `undefined`.